### PR TITLE
Bump jackson to 2.10.0

### PR DESCRIPTION
**Pull request by gastaldi**
Release notes: https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.10

### CODE REVIEW

**Comment by gastaldi** [comment]
According to https://twitter.com/fasterxml/status/1177385947933003776 looks like there were some problems with OSSRH while uploading the secondary artifacts. 

Hold the merge until this is resolved

**Comment by cescoffier** [comment]
I would prefer to check with @vietj to be sure there is no incompatibility with Vert.x.

**Comment by loicmathieu** [comment]
I made the update to 2.9.10 a few days ago and I just discover that I miss the dependency on the integration test.

Maybe add a comment in both places that the Jackson version is duplicated and must be updated on both places.

File: integration-tests/jackson/model/pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <name>Quarkus - Integration Tests - Jackson - model</name>
 
     <properties>
-        <jackson.version>2.9.9.20190807</jackson.version>
+        <jackson.version>2.10.0</jackson.version>
```
Comment (gsmet) [inline integration-tests/jackson/model/pom.xml]: Can't we get rid of that one? I don't see why it would be a good idea to have that.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: integration-tests/jackson/model/pom.xml (hunk 1 above)
Comment (gastaldi) [reply integration-tests/jackson/model/pom.xml]: Done

**Comment by gastaldi** [comment]
OSSRH issues were resolved, now waiting for @vietj to confirm there are no incompatibility with Vert.x 3.8.x

**Comment by gastaldi** [comment]
Vert.x PR created: https://github.com/vert-x3/vertx-dependencies/pull/31

**Comment by gastaldi** [comment]
Rebased

**Comment by loicmathieu** [comment]
Jackson 2.9.10.1 just released an fixes some CVE: CVE-2019-16942, CVE-2019-16943, CVE-2019-17531

I'm pretty sure that as 2.10 has been released before those CVEs were found it is vulnerable to them, so my advise is to wait for some minor release of 2.10 to happens before merging it.

**Comment by gastaldi** [comment]
Agreed, closing this now

**Pull request closed without merging**